api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.dhsc.htbhf.claimant.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.dhsc.htbhf.claimant.entity.CardStatus;
+import uk.gov.dhsc.htbhf.eligibility.model.CombinedIdentityAndEligibilityResponse;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Value
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+@ApiModel(description = "An existing claim for Apply for healthy start.")
+public class ClaimDTO {
+
+    @ApiModelProperty(notes = "The claim's id as stored in the database", example = "96c3f8c0-f6d9-4ad4-9ed9-72fcbd8d692d")
```

Comment:
Not sure we should be mentioning the database, how about `The unique id of the claim`

Reply:
Updated.